src/reducers/locations.js

```diff
@@ -83,6 +91,60 @@ const locationsReducer = (state = {}, action) => {
           RequiredDocuments[0] = proof;
         }
 
+        if (area != null) {
+          ServiceAreas[0] = {
+            ...area,
+            updated_at: dateString,
+          };
+        }
+
+        if (membership) {
+          const membershipIdx = Eligibilities
```

Comment:
Nitpick, but any reason this isn't just a `find`?

Reply:
Because we pass the `Eligibilities` object at the end, so  if we use find, we will have to construct a new array

Reply:
Not sure why that would be needed. I mean, since objects are always just referenced in JS (including when they're in an array) and `find` doesn't create a copy of the object, it should work either way.

I mean:
```
> const a = [{ a: 1, b: 2 }, { a: 2 }]
> const foo = a.find(x => x.a === 2)
> foo.b = 3;
> a
[ { a: 1, b: 2 }, { a: 2, b: 3 } ]
```

But maybe I misunderstood your comment.